lib/modules/commentQuickCollapse.js

```diff
@@ -0,0 +1,37 @@
+/* @flow */
+
+import { $ } from '../vendor';
+import { Module } from '../core/module';
+
+export const module: Module<*> = new Module('commentQuickCollapse');
+
+module.moduleName = 'commentQuickCollapseName';
+module.category = 'commentsCategory';
+module.description = 'commentQuickCollapseDesc';
+
+module.options = {
+	hideCommentsOnHeaderDoubleClick: {
+		type: 'boolean',
+		value: true,
+		description: 'hideCommentsOnHeaderDoubleClickDesc',
+	},
+};
+
+module.go = () => {
+	if (module.options.hideCommentsOnHeaderDoubleClick.value) {
+		hideCommentsOnHeaderDoubleClick();
+	}
+};
+
+function hideCommentsOnHeaderDoubleClick() {
+	$('p.tagline').on('dblclick', (e: Event) => {
```

Comment:
How about `$(document.body).on('dblclick', '.tagline', ...)` to support comments loaded dynamically?